cg-producer/setup.cfg

```diff
@@ -0,0 +1,2 @@
+[metadata]
+description-file = pypi-readme.md
```

Comment:
Is it a common convention for this particular file name to not be capitalized? If not, I would suggest renaming it to `PyPI-README.md`.

Reply:
No, there is no such convention, changed the file name accordingly on [12c573e](https://github.com/fasten-project/pypi-tools/pull/6/commits/12c573e8ab02b3848bd09ca765d8f02a335a32c5)